pkg/metrics/bootstrap.go

```diff
@@ -30,28 +23,17 @@ func Bootstrap(ctx context.Context, namespace string, client client.Client) erro
 	defer span.End()
 	tracing.SetInstanceID(ctx, namespace)
 
-	config := prometheus.Config{
-		Registry: metrics.Registry.(*prometheusclient.Registry),
-	}
-	c := controller.New(
-		processor.New(
-			selector.NewWithHistogramDistribution(
-				histogram.WithExplicitBoundaries(config.DefaultHistogramBoundaries),
-			),
-			export.CumulativeExportKindSelector(),
-			processor.WithMemory(true),
-		),
-		controller.WithResource(resource.NewWithAttributes([]attribute.KeyValue{}...)),
-	)
```

Comment:
this is one of the most controversial changes I made. I looked through the whole code base and it looks that there are no traces of controllers, processors and selectors anymore. At the same time, I could not find a sensible equivalent or a place where these settings could be configured. A documentation about migrating from the old to the new API would've been great.

I'm not sure if they are not needed anymore or the new packages are using some defaults that are better suited and allow removing the need for configuration on client code.

Reply:
>  A documentation about migrating from the old to the new API would've been great. 

Unfortunately, I very much doubt it. Probably here we would have to follow the otel change log of the last year. :face_exhaling: 

> I'm not sure if they are not needed anymore or the new packages are using some defaults that are better suited and allow removing the need for configuration on client code. 

Ive here no good answer. I made a local tour and could not detect any abnormalities. @rubenvp8510 what are your thoughts?